src/components/Attachments/AttachmentCarousel/Pager/index.tsx

```diff
@@ -19,30 +20,48 @@ type AttachmentCarouselPagerHandle = {
     setPage: (selectedPage: number) => void;
 };
 
-type PagerItem = {
-    key: string;
-    url: string;
+type Attachment = {
     source: string;
 };
 
 type AttachmentCarouselPagerProps = {
-    items: PagerItem[];
-    renderItem: (props: {item: PagerItem; index: number; isActive: boolean}) => React.ReactNode;
-    initialIndex: number;
+    /**
+     * The attachments to be rendered in the pager.
+     */
```

Comment:
There is a mix of comment styles in this PR. The most consistent pattern across the codebase to use is to keep it all on a single like like `/** blah blah */`. Can you please go through this PR and make sure all the prop comments match this style?